newsroom/users/users.py

```diff
@@ -361,7 +362,7 @@ def check_permissions(self, doc, updates=None):
             if doc.get("company") and doc["company"] == manager.get("company"):
                 allowed_updates = (
                     COMPANY_ADMIN_ALLOWED_UPDATES
-                    if not get_setting("allow_companies_to_manage_products")
+                    if not await get_setting("allow_companies_to_manage_products")
```

Comment:
Through changing `get_setting` back to sync, this entire module would no longer require the `async` changes